.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/sql
\ No newline at end of file
+*   @anirudha @penghuo @ps48 @kavithacm @derek-ho @joshuali925 @YANG-DB @rupal-bq @mengweieric @vamsi-amazon, @Yury-Fridlyand, @MaxKsyunz
```

Comment:
looks like `,` is causing error. Can you please remove it?

Reply:
my b, done